careers.html

```diff
@@ -0,0 +1,143 @@
+    </div>
+</section>
+
+<section class="join-us" id="js-join-us">
+    <p class="section-heading">Join us</p>
+    <p class="heading-bottom-line"></p>
+    <p class="section-subline">If you like getting things done, Infratab is the place for you. We are always looking for people who want to make a difference with their work.</p>
+    <div class="row">
+        <div class="col l3">
+            <div class="hiring-box">
+                <p class="hiring-heading">Frontend Developer</p>
+                <p class="hiring-require">Experience : 2+ years</p>
+                <p class="hiring-require">Skills : BackboneJs, ReactJs, Redux, React</p>
+                <p class="hiring-button">Apply</p>
+            </div>
+        </div>
+        <div class="col l3">
+            <div class="hiring-box">
+                <p class="hiring-heading">UI/UX Designer</p>
+                <p class="hiring-require">Experience : 2+ years</p>
+                <p class="hiring-require">Skills : Sketch, PhotoShop, Illustratore, HTML/CSS</p>
+                <p class="hiring-button">Apply</p>
+            </div>
+        </div>
+        <div class="col l3">
+            <div class="hiring-box">
+                <p class="hiring-heading">Backend Developer</p>
+                <p class="hiring-require">Experience : 2+ years</p>
+                <p class="hiring-require">Skills : Python, DJango, Postgresql, DRF</p>
+                <p class="hiring-button">Apply</p>
+            </div>
+        </div>
+        <div class="col l3">
+            <div class="hiring-box">
+                <p class="hiring-heading">Data Engineer</p>
+                <p class="hiring-require">Experience : 2+ years</p>
+                <p class="hiring-require">Skills : Python, MySQL, Data Analytics, Django</p>
+                <p class="hiring-button">Apply</p>
+            </div>
+        </div>
+    </div>
+    <p class="join-us-cv">Looking for a different role?</br>
+    Drop us a message with your CV at <a href="contact.html" class="contact-link">here</a> and we will get back to you.</p>
+</section>
+
+<footer class="footer">
+  <img src="img/waves.png" class="wave-image">
+  <div class="footer-content">
+    <div class="row">
+      <div class="col l2 m2 s2">
+        <p class="footer-heading">Company</p>
+        <a href=# class="footer-link">Home</a>
+        <a href=# class="footer-link">Careers</a>
+        <a href=# class="footer-link">Contact</a>
+      </div>
+      <div class="col l8 m8 s8 center">
+        <img src="img/infratab-logo.png" class="footer-logo">
+      </div>
+      <div class="col l2 m2 s2">
+         <p class="footer-heading">Contact</p>
+         <a class="footer-link"><i class="fa fa-envelope"></i> : info@infratab.in</a>
+         <a class="footer-link"><i class="fa fa-phone""></i> : (+91) 80 4092 9250</a>
+         <a class="footer-link"><i class="fa fa-globe"></i> : www.infratab.com</a>
+      </div>
+    </div>
+    <p class="horizontal-line"></p>
```

Comment:
@priyanka0901 Why `p` tag for adding a horizontal line. There is two other way to do it adding `<hr>` tag or else border-bottom from the content container or else border-top from copyright section.